src/utils.ts

```diff
@@ -40,4 +45,81 @@ export function fixRemoteUrl(remoteAddress: string, remotePort: number, target:
         }
     }
     return target;
+}
+
+
+export const enum Platform {
+    Windows, OSX, Linux
+}
+
+export function getPlatform(): Platform {
+    const platform = os.platform();
+    return platform === 'darwin' ? Platform.OSX :
+        platform === 'win32' ? Platform.Windows :
+            Platform.Linux;
+}
+
+export function existsSync(path: string): boolean {
+    try {
+        fs.statSync(path);
+        return true;
+    } catch (e) {
+        return false;
+    }
+}
+
+export function launchLocalChrome(targetUrl: string) {
+    const chromePath = getPathToChrome();
+    const chromeArgs = ['--remote-debugging-port=9222'];
+
+    const chromeProc = cp.spawn(chromePath, chromeArgs, {
+        stdio: 'ignore',
+        detached: true
+    });
+
+    chromeProc.unref();
+}
+
+export async function isPortFree(host: string, port: number): Promise<boolean> {
+    return new Promise<boolean>((resolve) => {
+        const server = net.createServer();
+
+        server.on('error', () => resolve(false));
+        server.listen(port, host);
+
+        server.on('listening', () => {
+            server.close();
+            server.unref();
+        });
+
+        server.on('close', () => resolve(true));
+    });
+}
+
+const WIN_APPDATA = process.env.LOCALAPPDATA || '/';
+const DEFAULT_CHROME_PATH = {
+    LINUX: '/usr/bin/google-chrome',
+    OSX: '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome',
+    WIN: 'C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe',
+    WIN_LOCALAPPDATA: path.join(WIN_APPDATA, 'Google\\Chrome\\Application\\chrome.exe'),
+    WINx86: 'C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe',
+};
+
+export function getPathToChrome(): string {
+    const platform = getPlatform();
```

Comment:
Should you be able to override these with a custom setting?

Reply:
updated.